tests/unit/models/test_evc_base.py

```diff
@@ -37,7 +37,9 @@ def test_expected_requiring_redeploy_attributes(self) -> None:
             "queue_id",
             "sb_priority",
             "primary_constraints",
-            "secondary_constraints"
+            "secondary_constraints",
+            "uni_a",
+            "uni_z"
```

Comment:
```suggestion
            "uni_z",
```

Minor suggestion to make it easy in an eventual next modification.